assignment/tool/src/webapp/js/assignments.js

```diff
@@ -852,6 +852,24 @@ ASN.toggleAutoAnnounceOptions = function(checked){
     }
 };
 
+ASN.toggleAutoAnnounceEstimate = function(checked){
+    var obEstimate = document.getElementById("checkobestimate");
```

Comment:
Stick to a certain number of spaces for tabs. I prefer 2, but it depends on the file I'm working on.

Reply:
The whole branch has been updated to have 2 spaces instead of tabs.